library/src/main/java/com/bumptech/glide/load/engine/cache/DiskLruCacheWrapper.java

```diff
@@ -114,4 +114,16 @@ public void delete(Key key) {
             }
         }
     }
+
+    @Override
+    public void clear() {
+        try {
+            getDiskCache().delete();
+            diskLruCache = null;
```

Comment:
I'd guess that findbugs (the reason for the build breakage) is complaining about the unsynchronized access to diskLruCache here.